mlflow/environment_variables.py

```diff
@@ -0,0 +1,44 @@
+"""
+This module defines environment variable names used in MLflow.
+"""
+import os
+
+
+class _EnvironmentVariable:
+    """
+    Define a environment variable
+    """
+    def __init__(self, name, type, default):
+        self.name = name
+        self.type = type
+        self.default = default
+
+    def get(self):
+        """
+        Get environment variable value.
+        """
```

Comment:
```suggestion
        """
        Reads the environment variable if it exists and casts it to the specified type. Otherwise, returns the default value.
        """
```